package.json

```diff
@@ -1,11 +1,16 @@
 {
   "dependencies": {
+    "babel-runtime": "^6.26.0",
```

Comment:
This is under `dependencies` instead of `devDependencies` to prepare for the future where we will use Babel for all Sylius JS files. :laughing: